packages/components/.storybook/Button/button.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+
+import Clickable, { ClickableProps } from "../../src/util/clickable";
+
+import { ButtonProps } from "../../src/Button";
+import Heading from "../../src/Heading";
+import Text from "../../src/Text";
+import styles from "./button.module.css";
+
+export { default } from "../../src/Button/button.stories";
```

Comment:
very cool how this keeps them all in the same storybook folder!

Reply:
Right! I had a hunch it might, but I was really happy to be proven right 😄